tcms/kiwi_auth/tests/test_admin.py

```diff
@@ -188,3 +191,33 @@ def test_should_be_able_to_edit_a_non_default_group(self):
             '<input type="text" name="name" value="%s" class="vTextField"'
             ' maxlength="150" required id="id_name">' % self.group.name,
         )
+
+    def test_should_be_allowed_to_create_new_group_with_added_user(self):
+        group_name = "TestGroupName"
+        response = self.client.post(
+            reverse("admin:auth_group_add"),
+            {"name": group_name, "users": [self.tester.id]},
+            follow=True,
+        )
+
+        self.assertContains(response, group_name)
+        self.assertContains(response, "was added successfully.")
+        self.assertTrue(self.tester.groups.filter(name=group_name).exists())
+
+    def test_should_be_able_to_add_user_while_editing_a_group(self):
+        self.client.post(
+            reverse("admin:auth_group_change", args=[self.group.id]),
+            {"name": self.group.name, "users": [self.tester.id]},
+            follow=True,
+        )
+
+        response = self.client.get(
+            reverse("admin:auth_group_change", args=[self.group.id])
+        )
+
+        self.assertContains(
+            response,
+            '<option value="%s" selected>%s</option>'
+            % (self.tester.pk, self.tester.username),
+        )
+        self.assertTrue(self.tester.groups.filter(name=self.group.name).exists())
```

Comment:
I think the transactions work fine atm but to be on the safe side I'd like to see an `assertFalse` assertion as the first thing in all scenarios which have this assertion. (or in a setUp() method if you want). This will make sure we're always starting with clean test data.